setup.py

```diff
@@ -190,6 +190,15 @@
     ],
 }
 
+extras_require = {}
+_extras_groups = [
+    ('requirements', 'requirements.txt'), ('requirements-py2', 'requirements-py2.txt'),
+    ('setuptools', 'requirement-setuptools.txt'), ('dev', 'dev-requirements.txt'),
+]
+for group, filepath in _extras_groups:
+    with open(filepath, 'r') as f:
```

Comment:
this assumes you're running setup.py from the same working directory. better to use the path of the current module to be safe

Reply:
Updated. I'm usually using `pip install -e ...` which implicitly changes directory to the package root before installation, so I didn't notice the problem, even though I've installed it from outside of CKAN's repo.